# Introduction 
With the use of high school data compiled from the New York City Department of Education, and the SAT score averages and testing rates provided by the College Board, we analyse and discuss whether a student's high school and borrough affects the SAT score he gets through college board. SAT scores are standardized tests taken by the college board (an independent board) that are used by students to apply for college after highschool. The score a child gets greatly impacts his/her chance in getting into college and their future. Though the new SAT tests have changed their format to 1600 points (Math: 800, Reading: 400, Writing: 400), however, we will be using data from the older format which gives us almost the same results (Math: 800, Reading: 800, Writing: 800). The main emphasis in the project is to find connections in whether some boroughs or highschools in New York City do better than others and discuss the factors that lead to these differences. Finding differences would be extremely interesting as one can infer how getting into certain public highschools or going for schooling in certain areas of the city would impact the college a child gets into. 


Our data is very comprehensive, and to answer our question: (Y Variable) whether borough or high school impact the SAT score a child gets, we will use different variables. One of our variable (X_1) will be the borough Manhattan, the second will be Average SAT Reading Scores (X_2), Average SAT Math Scores (X_3), Average SAT Reading Scores (X_4). With the use of these variables we will try to develop an analysis on whether the borough you live in around NYC would give certain students a natural advantage. 

In [1]:
import pandas as pd
import qeds
%matplotlib inline
# activate plot theme
import qeds
qeds.themes.mpl_style();

from IPython.display import display
import ssl
ssl._create_default_https_context = ssl._create_unverified_context

First we are Reading the data and creating a dataframe 'df' and then running our dataframe. 

In [2]:
df = pd.read_csv(r"C:\Users\Anwar Malik\Desktop\UofT\Year 2\ECO225\ECO225Project\Data\scores.csv")
df

School ID                                        School Name    Borough  \
0      02M260                 Clinton School Writers and Artists  Manhattan   
1      06M211  Inwood Early College for Health and Informatio...  Manhattan   
2      01M539  New Explorations into Science, Technology and ...  Manhattan   
3      02M294                               Essex Street Academy  Manhattan   
4      02M308                       Lower Manhattan Arts Academy  Manhattan   
..        ...                                                ...        ...   
430    27Q302  Queens High School for Information, Research, ...     Queens   
431    27Q324  Rockaway Park High School for Environmental Su...     Queens   
432    27Q262                   Channel View School for Research     Queens   
433    27Q351                    Rockaway Collegiate High School     Queens   
434    27Q323                                  Scholars' Academy     Queens   

    Building Code              Street Address           City State  Zip Code  \
0            M933        425 West 33rd Street      Manhattan    NY     10001   
1            M052          650 Academy Street      Manhattan    NY     10002   
2            M022         111 Columbia Street      Manhattan    NY     10002   
3            M445            350 Grand Street      Manhattan    NY     10002   
4            M445            350 Grand Street      Manhattan    NY     10002   
..            ...                         ...            ...   ...       ...   
430          Q465        8-21 Bay 25th Street   Far Rockaway    NY     11691   
431          Q410  100-00 Beach Channel Drive  Rockaway Park    NY     11694   
432          Q410  100-00 Beach Channel Drive  Rockaway Park    NY     11694   
433          Q410  100-00 Beach Channel Drive  Rockaway Park    NY     11694   
434          Q180      320 Beach 104th Street  Rockaway Park    NY     11694   

     Latitude  Longitude  ... End Time Student Enrollment Percent White  \
0    40.75321  -73.99786  ...      NaN                NaN           NaN   
1    40.86605  -73.92486  ...  3:00 pm               87.0         3.40%   
2    40.71873  -73.97943  ...  4:00 pm             1735.0        28.60%   
3    40.71687  -73.98953  ...  2:45 pm              358.0        11.70%   
4    40.71687  -73.98953  ...  3:00 pm              383.0         3.10%   
..        ...        ...  ...      ...                ...           ...   
430  40.60199  -73.76283  ...  4:10 pm              381.0         2.10%   
431  40.58601  -73.82309  ...  3:30 pm              368.0        14.70%   
432  40.58601  -73.82309  ...  3:04 pm              683.0        10.70%   
433  40.58601  -73.82309  ...  4:20 pm              397.0         3.80%   
434  40.58413  -73.82568  ...  3:00 pm             1302.0        34.70%   

     Percent Black Percent Hispanic Percent Asian Average Score (SAT Math)  \
0              NaN              NaN           NaN                      NaN   
1           21.80%           67.80%         4.60%                      NaN   
2           13.30%           18.00%        38.50%                    657.0   
3           38.50%           41.30%         5.90%                    395.0   
4           28.20%           56.90%         8.60%                    418.0   
..             ...              ...           ...                      ...   
430         49.10%           43.60%         4.20%                    372.0   
431         46.70%           30.20%         3.50%                    357.0   
432         53.20%           29.10%         6.00%                    427.0   
433         52.60%           33.50%         5.00%                    399.0   
434         22.70%           19.60%        21.40%                    588.0   

    Average Score (SAT Reading)  Average Score (SAT Writing)  Percent Tested  
0                           NaN                          NaN             NaN  
1                           NaN                          NaN             NaN  
2                  

# Data Cleaning
To start our analysis we need to do certain steps to clean our data and convert it into something that we will find easier to interpret. Since we care about SAT scores and the distribution of different ethnic groups in each high school, we will need to clean this data and remove the '%' sign and convert all percentages and SAT scores in to float. 

In the cell below we clean the data remove the '%' sign from the data. Using string methods we removed the percentage sign. 

In [3]:
df2 = df.replace('%','',regex=True)

In [4]:
df2

School ID                                        School Name    Borough  \
0      02M260                 Clinton School Writers and Artists  Manhattan   
1      06M211  Inwood Early College for Health and Informatio...  Manhattan   
2      01M539  New Explorations into Science, Technology and ...  Manhattan   
3      02M294                               Essex Street Academy  Manhattan   
4      02M308                       Lower Manhattan Arts Academy  Manhattan   
..        ...                                                ...        ...   
430    27Q302  Queens High School for Information, Research, ...     Queens   
431    27Q324  Rockaway Park High School for Environmental Su...     Queens   
432    27Q262                   Channel View School for Research     Queens   
433    27Q351                    Rockaway Collegiate High School     Queens   
434    27Q323                                  Scholars' Academy     Queens   

    Building Code              Street Address           City State  Zip Code  \
0            M933        425 West 33rd Street      Manhattan    NY     10001   
1            M052          650 Academy Street      Manhattan    NY     10002   
2            M022         111 Columbia Street      Manhattan    NY     10002   
3            M445            350 Grand Street      Manhattan    NY     10002   
4            M445            350 Grand Street      Manhattan    NY     10002   
..            ...                         ...            ...   ...       ...   
430          Q465        8-21 Bay 25th Street   Far Rockaway    NY     11691   
431          Q410  100-00 Beach Channel Drive  Rockaway Park    NY     11694   
432          Q410  100-00 Beach Channel Drive  Rockaway Park    NY     11694   
433          Q410  100-00 Beach Channel Drive  Rockaway Park    NY     11694   
434          Q180      320 Beach 104th Street  Rockaway Park    NY     11694   

     Latitude  Longitude  ... End Time Student Enrollment Percent White  \
0    40.75321  -73.99786  ...      NaN                NaN           NaN   
1    40.86605  -73.92486  ...  3:00 pm               87.0          3.40   
2    40.71873  -73.97943  ...  4:00 pm             1735.0         28.60   
3    40.71687  -73.98953  ...  2:45 pm              358.0         11.70   
4    40.71687  -73.98953  ...  3:00 pm              383.0          3.10   
..        ...        ...  ...      ...                ...           ...   
430  40.60199  -73.76283  ...  4:10 pm              381.0          2.10   
431  40.58601  -73.82309  ...  3:30 pm              368.0         14.70   
432  40.58601  -73.82309  ...  3:04 pm              683.0         10.70   
433  40.58601  -73.82309  ...  4:20 pm              397.0          3.80   
434  40.58413  -73.82568  ...  3:00 pm             1302.0         34.70   

     Percent Black Percent Hispanic Percent Asian Average Score (SAT Math)  \
0              NaN              NaN           NaN                      NaN   
1            21.80            67.80          4.60                      NaN   
2            13.30            18.00         38.50                    657.0   
3            38.50            41.30          5.90                    395.0   
4            28.20            56.90          8.60                    418.0   
..             ...              ...           ...                      ...   
430          49.10            43.60          4.20                    372.0   
431          46.70            30.20          3.50                    357.0   
432          53.20            29.10          6.00                    427.0   
433          52.60            33.50          5.00                    399.0   
434          22.70            19.60         21.40                    588.0   

    Average Score (SAT Reading)  Average Score (SAT Writing)  Percent Tested  
0                           NaN                          NaN             NaN  
1                           NaN                          NaN             NaN  
2                  

Our emphasis is to determine whether certain boroughs may have higher SAT scores, as a result, we calulate the percentage of each borough in our data.

In [5]:
df2['Borough'].value_counts()

Brooklyn         121
Bronx            118
Manhattan        106
Queens            80
Staten Island     10
Name: Borough, dtype: int64

In [6]:
df2['Borough'].value_counts(normalize= True)

Brooklyn         0.278161
Bronx            0.271264
Manhattan        0.243678
Queens           0.183908
Staten Island    0.022989
Name: Borough, dtype: float64

We will now remove any rows that contain missing data. This will allow us to interpret our data much easily when we run our analysis. 

In [13]:
df2.dropna()

School ID                                        School Name    Borough  \
2      01M539  New Explorations into Science, Technology and ...  Manhattan   
3      02M294                               Essex Street Academy  Manhattan   
4      02M308                       Lower Manhattan Arts Academy  Manhattan   
5      02M545    High School for Dual Language and Asian Studies  Manhattan   
6      01M292      Henry Street School for International Studies  Manhattan   
..        ...                                                ...        ...   
430    27Q302  Queens High School for Information, Research, ...     Queens   
431    27Q324  Rockaway Park High School for Environmental Su...     Queens   
432    27Q262                   Channel View School for Research     Queens   
433    27Q351                    Rockaway Collegiate High School     Queens   
434    27Q323                                  Scholars' Academy     Queens   

    Building Code              Street Address           City State  Zip Code  \
2            M022         111 Columbia Street      Manhattan    NY     10002   
3            M445            350 Grand Street      Manhattan    NY     10002   
4            M445            350 Grand Street      Manhattan    NY     10002   
5            M445            350 Grand Street      Manhattan    NY     10002   
6            M056            220 Henry Street      Manhattan    NY     10002   
..            ...                         ...            ...   ...       ...   
430          Q465        8-21 Bay 25th Street   Far Rockaway    NY     11691   
431          Q410  100-00 Beach Channel Drive  Rockaway Park    NY     11694   
432          Q410  100-00 Beach Channel Drive  Rockaway Park    NY     11694   
433          Q410  100-00 Beach Channel Drive  Rockaway Park    NY     11694   
434          Q180      320 Beach 104th Street  Rockaway Park    NY     11694   

     Latitude  Longitude  ... End Time Student Enrollment Percent White  \
2    40.71873  -73.97943  ...  4:00 pm             1735.0         28.60   
3    40.71687  -73.98953  ...  2:45 pm              358.0         11.70   
4    40.71687  -73.98953  ...  3:00 pm              383.0          3.10   
5    40.71687  -73.98953  ...  3:35 pm              416.0          1.70   
6    40.71376  -73.98526  ...  3:30 pm              255.0          3.90   
..        ...        ...  ...      ...                ...           ...   
430  40.60199  -73.76283  ...  4:10 pm              381.0          2.10   
431  40.58601  -73.82309  ...  3:30 pm              368.0         14.70   
432  40.58601  -73.82309  ...  3:04 pm              683.0         10.70   
433  40.58601  -73.82309  ...  4:20 pm              397.0          3.80   
434  40.58413  -73.82568  ...  3:00 pm             1302.0         34.70   

     Percent Black Percent Hispanic Percent Asian Average Score (SAT Math)  \
2            13.30            18.00         38.50                    657.0   
3            38.50            41.30          5.90                    395.0   
4            28.20            56.90          8.60                    418.0   
5             3.10             5.50         88.90                    613.0   
6            24.40            56.60         13.20                    410.0   
..             ...              ...           ...                      ...   
430          49.10            43.60          4.20                    372.0   
431          46.70            30.20          3.50                    357.0   
432          53.20            29.10          6.00                    427.0   
433          52.60            33.50          5.00                    399.0   
434          22.70            19.60         21.40                    588.0   

    Average Score (SAT Reading)  Average Score (SAT Writing)  Percent Tested  
2                         601.0                        601.0           91.00  
3                         411.0                        387.0           78.90  
4                  

Converting all our new variables that are without the % sign to floats since they were previosuly as strings. We also make a new variable 'Non-White' which includes the Black, Hispanic and Asian population. 

In [ ]:
df2['Percent White'] = df2['Percent White'].astype(float)
df2['Percent Black'] = df2['Percent Black'].astype(float)
df2['Percent Hispanic'] = df2['Percent Hispanic'].astype(float)
df2['Percent Asian'] = df2['Percent Asian'].astype(float)
df2['Non-White'] = df2['Percent Black'] +  df2['Percent Hispanic'] + df2['Percent Asian']
df2['Percent Tested'] = df2['Percent Tested'].astype(float)

# Summary Statistics 
Part of our analysis is to determine whether certain boroughs have lower SAT scores than others. An important inferance we hope to make is whether regions with higher immigrated or non-white populations might be at a disadvantage and getting lower SAT scores. The reasons could be poor infrastructure, lack of resources and equal opportunities. Hence, we will analyse and create a summary statistics.

As a result, we will use the data we have of percentage of each populations and the average SAT scores to see whether there is correlation between a borrough with a high white population and a low white population. 

In [32]:
df2['Total Average Score'] = df2['Average Score (SAT Math)'] + df2['Average Score (SAT Reading)'] + df2['Average Score (SAT Writing)']

In [34]:
Manhattan_SAT = ['Manhattan']
df_manhattan = df2[df2['Borough'].isin(Manhattan_SAT)]
df_manhattan.describe()

Zip Code   Latitude  Longitude  Student Enrollment  Percent White  \
count     89.000000  89.000000  89.000000           89.000000      89.000000   
mean   10024.438202  40.764681 -73.976222          638.089888      10.108989   
std       33.271253   0.042536   0.024753          495.258028      14.282440   
min    10002.000000  40.690790 -74.019760          234.000000       0.000000   
25%    10009.000000  40.732490 -73.991480          413.000000       1.900000   
50%    10019.000000  40.765030 -73.984820          472.000000       3.500000   
75%    10030.000000  40.791880 -73.954010          629.000000      10.800000   
max    10282.000000  40.861110 -73.927030         3296.000000      61.400000   

       Percent Black  Percent Hispanic  Percent Asian  \
count      89.000000         89.000000      89.000000   
mean       25.425843         51.191011      11.173034   
std        15.423856         21.621368      15.399120   
min         0.000000          2.600000       0.000000   
25%        14.200000         40.100000       2.200000   
50%        24.500000         53.800000       5.800000   
75%        32.500000         62.100000      13.700000   
max        77.600000        100.000000      88.900000   

       Average Score (SAT Math)  Average Score (SAT Reading)  \
count                 89.000000                    89.000000   
mean                 455.887640                   444.932584   
std                   83.818315                    73.930987   
min                  344.000000                   330.000000   
25%                  395.000000                   395.000000   
50%                  433.000000                   428.000000   
75%                  485.000000                   468.000000   
max                  754.000000                   697.000000   

       Average Score (SAT Writing)  Percent Tested   Non-White  \
count                    89.000000       89.000000   89.000000   
mean                    439.314607       71.197753   87.789888   
std                      76.387707       17.740426   15.274459   
min                     316.000000       30.600000   34.800000   
25%                     387.000000       58.600000   87.100000   
50%                     415.000000       71.800000   94.500000   
75%                     463.000000       87.000000   96.500000   
max                     693.000000      100.000000  100.000000   

       Total_Average_Score  Total Average Score  
count            89.000000            89.000000  
mean           1340.134831          1340.134831  
std             230.294140           230.294140  
min            1005.000000          1005.000000  
25%            1173.000000          1173.000000  
50%            1284.000000          1284.000000  
75%            1415.000000          1415.000000  
max            2144.000000          2144.000000

In [35]:
Brooklyn_SAT = ['Brooklyn']
df_brooklyn = df2[df2['Borough'].isin(Brooklyn_SAT)]
df_brooklyn.describe()

Zip Code    Latitude   Longitude  Student Enrollment  \
count    109.000000  109.000000  109.000000          109.000000   
mean   11216.211009   40.665302  -73.947209          742.550459   
std       11.631752    0.035739    0.037010          919.736088   
min    11201.000000   40.580850  -74.039950          142.000000   
25%    11207.000000   40.648660  -73.979260          331.000000   
50%    11214.000000   40.670300  -73.953240          438.000000   
75%    11225.000000   40.694480  -73.921080          605.000000   
max    11238.000000   40.722640  -73.868430         5447.000000   

       Percent White  Percent Black  Percent Hispanic  Percent Asian  \
count     109.000000     109.000000        109.000000     109.000000   
mean        7.177064      54.989908         28.790826       7.447706   
std        11.832548      29.353765         22.798658      11.823420   
min         0.200000       0.000000          3.300000       0.000000   
25%         0.900000      25.700000         12.500000       1.100000   
50%         2.000000      67.600000         21.900000       2.500000   
75%         6.400000      82.100000         36.500000       6.600000   
max        59.100000      91.200000         99.600000      60.500000   

       Average Score (SAT Math)  Average Score (SAT Reading)  \
count                 109.00000                   109.000000   
mean                  416.40367                   410.761468   
std                    60.48142                    48.914684   
min                   319.00000                   302.000000   
25%                   379.00000                   381.000000   
50%                   395.00000                   398.000000   
75%                   441.00000                   427.000000   
max                   682.00000                   608.000000   

       Average Score (SAT Writing)  Percent Tested   Non-White  \
count                   109.000000      109.000000  109.000000   
mean                    403.091743       62.423853   91.228440   
std                      49.657072       17.290208   12.052471   
min                     284.000000       18.500000   39.500000   
25%                     378.000000       50.800000   91.600000   
50%                     394.000000       61.600000   96.500000   
75%                     417.000000       72.700000   97.700000   
max                     606.000000       98.100000   99.600000   

       Total_Average_Score  Total Average Score  
count           109.000000           109.000000  
mean           1230.256881          1230.256881  
std             154.868427           154.868427  
min             926.000000           926.000000  
25%            1141.000000          1141.000000  
50%            1186.000000          1186.000000  
75%            1298.000000          1298.000000  
max            1896.000000          1896.000000

In [36]:
Bronx_SAT = ['Bronx']
df_bronx = df2[df2['Borough'].isin(Bronx_SAT)]
df_bronx.describe()

Zip Code   Latitude  Longitude  Student Enrollment  Percent White  \
count     98.000000  98.000000  98.000000           98.000000      98.000000   
mean   10461.979592  40.845445 -73.889419          541.724490       3.121429   
std        6.904323   0.022944   0.023894          386.790355       6.156469   
min    10451.000000  40.810040 -73.930320          170.000000       0.000000   
25%    10457.000000  40.827593 -73.908028          385.250000       0.925000   
50%    10460.000000  40.840370 -73.895300          449.500000       1.500000   
75%    10468.000000  40.870380 -73.867620          552.000000       2.575000   
max    10475.000000  40.888370 -73.833800         3015.000000      50.800000   

       Percent Black  Percent Hispanic  Percent Asian  \
count      98.000000         98.000000      98.000000   
mean       28.974490         62.711224       3.905102   
std        12.712526         14.907425       7.011954   
min         0.000000          5.500000       0.000000   
25%        23.225000         56.825000       1.025000   
50%        28.750000         65.100000       2.250000   
75%        34.250000         70.375000       3.950000   
max        63.600000        100.000000      62.800000   

       Average Score (SAT Math)  Average Score (SAT Reading)  \
count                 98.000000                    98.000000   
mean                 404.357143                   402.520408   
std                   52.215898                    49.224340   
min                  317.000000                   311.000000   
25%                  378.250000                   381.000000   
50%                  395.500000                   400.500000   
75%                  418.000000                   417.000000   
max                  714.000000                   672.000000   

       Average Score (SAT Writing)  Percent Tested   Non-White  \
count                    98.000000       98.000000   98.000000   
mean                    395.846939       58.220408   95.590816   
std                      51.304298       18.684036    6.935195   
min                     292.000000       19.200000   42.300000   
25%                     371.500000       44.225000   96.125000   
50%                     391.500000       57.850000   97.200000   
75%                     409.750000       73.600000   97.975000   
max                     672.000000       97.000000  100.000000   

       Total_Average_Score  Total Average Score  
count            98.000000            98.000000  
mean           1202.724490          1202.724490  
std             150.393901           150.393901  
min             924.000000           924.000000  
25%            1131.000000          1131.000000  
50%            1190.000000          1190.000000  
75%            1245.000000          1245.000000  
max            2041.000000          2041.000000

In [37]:
Queens_SAT = ['Queens']
df_queens = df2[df2['Borough'].isin(Queens_SAT)]
df_queens.describe()

Zip Code   Latitude  Longitude  Student Enrollment  Percent White  \
count     68.000000  68.000000  68.000000           68.000000      68.000000   
mean   11355.911765  40.711839 -73.835470         1082.926471      10.920588   
std      173.413124   0.049238   0.067358          924.631420      11.341781   
min    11101.000000  40.584130 -73.949690          368.000000       0.000000   
25%    11292.000000  40.695540 -73.909070          504.750000       2.650000   
50%    11380.000000  40.725060 -73.822450          630.000000       5.850000   
75%    11432.000000  40.744948 -73.786255         1174.500000      15.500000   
max    11694.000000  40.771870 -73.726910         4172.000000      50.300000   

       Percent Black  Percent Hispanic  Percent Asian  \
count      68.000000         68.000000        68.0000   
mean       28.980882         34.377941        23.5000   
std        26.747712         21.026741        16.7086   
min         0.000000          5.700000         0.3000   
25%         7.250000         17.375000         9.6250   
50%        15.400000         31.800000        22.3500   
75%        49.350000         48.325000        34.2750   
max        89.700000         99.700000        75.4000   

       Average Score (SAT Math)  Average Score (SAT Reading)  \
count                 68.000000                    68.000000   
mean                 461.558824                   442.647059   
std                   72.111933                    61.959005   
min                  340.000000                   320.000000   
25%                  414.500000                   407.000000   
50%                  446.000000                   427.500000   
75%                  489.250000                   464.000000   
max                  701.000000                   640.000000   

       Average Score (SAT Writing)  Percent Tested   Non-White  \
count                    68.000000       68.000000   68.000000   
mean                    439.220588       68.980882   86.858824   
std                      66.514244       19.683710   11.327924   
min                     318.000000       26.300000   47.600000   
25%                     400.750000       54.525000   81.150000   
50%                     423.500000       71.400000   91.950000   
75%                     464.750000       84.875000   94.225000   
max                     661.000000      100.000000  100.000000   

       Total_Average_Score  Total Average Score  
count            68.000000            68.000000  
mean           1343.426471          1343.426471  
std             195.953796           195.953796  
min             978.000000           978.000000  
25%            1217.000000          1217.000000  
50%            1288.000000          1288.000000  
75%            1427.500000          1427.500000  
max            1981.000000          1981.000000

In [42]:
Staten_island_SAT = ['Staten Island']
df_staten_island = df2[df2['Borough'].isin(Staten_island_SAT)]
df_staten_island.describe()

Zip Code   Latitude  Longitude  Student Enrollment  Percent White  \
count     10.000000  10.000000  10.000000           10.000000      10.000000   
mean   10307.100000  40.595678 -74.125725         1846.000000      45.770000   
std        5.839521   0.037448   0.036464         1246.290228      20.156832   
min    10301.000000  40.528230 -74.192150          479.000000      17.700000   
25%    10301.250000  40.572390 -74.153915          779.750000      28.200000   
50%    10306.000000  40.590335 -74.119230         1477.500000      50.800000   
75%    10313.500000  40.627243 -74.102935         2718.000000      54.750000   
max    10314.000000  40.644740 -74.078710         3982.000000      79.900000   

       Percent Black  Percent Hispanic  Percent Asian  \
count      10.000000         10.000000      10.000000   
mean       16.780000         25.210000      10.740000   
std        12.563421         12.507105      11.075218   
min         1.000000          5.200000       3.300000   
25%         9.325000         19.200000       5.550000   
50%        12.950000         23.600000       7.050000   
75%        26.500000         33.950000      10.275000   
max        38.400000         42.800000      41.100000   

       Average Score (SAT Math)  Average Score (SAT Reading)  \
count                 10.000000                    10.000000   
mean                 486.200000                   478.500000   
std                   83.132825                    66.850164   
min                  420.000000                   427.000000   
25%                  444.750000                   449.000000   
50%                  465.500000                   463.000000   
75%                  491.250000                   475.250000   
max                  711.000000                   660.000000   

       Average Score (SAT Writing)  Percent Tested  Non-White  \
count                     10.00000       10.000000  10.000000   
mean                     474.30000       67.390000  52.730000   
std                       72.67133       18.612687  20.460425   
min                      409.00000       38.000000  18.700000   
25%                      444.00000       57.775000  41.675000   
50%                      459.00000       65.250000  48.400000   
75%                      474.50000       79.125000  71.100000   
max                      670.00000       99.700000  81.500000   

       Total_Average_Score  Total Average Score  
count            10.000000            10.000000  
mean           1439.000000          1439.000000  
std             222.303596           222.303596  
min            1258.000000          1258.000000  
25%            1346.500000          1346.500000  
50%            1382.000000          1382.000000  
75%            1441.000000          1441.000000  
max            2041.000000          2041.000000

As we run 5 different summary statistics of each of our 5 boroughs, we can make initial comments on how the SAT scores vary among the boroughs. Regions that tend to have higher white populations such as Staten Island, Queens and Manhattan have done considerably better an acheiving higher SAT scores which are over 1350 on average. Other regions such as Bronx and Brooklyn tend to have SAT scores close to 1200. During a very competitive college application period where each point matters, a difference of 150 points shows how extremely distributed SAT scores can be within a single city. 